frontend/src/Main/components/Cart.jsx

```
@@ -147,21 +148,23 @@ function Cart({ cart, removeFromCart, removeAllFromCart, checkoutCart }) {
-              (total, item) => total.add(item.totalGrossPrice),
-              new Decimal(0),
-            ),
-          )}
-        {checkoutProcessing && <Spinner color="gray" className="ml-3" />}
-      </MyButton>
+      {paymentMethods.map((paymentMethod) => (
+        <MyButton
+          {...((cart.length === 0 || checkoutProcessing) && { disabled: true })}
+          className="w-full mt-2 uppercase"
+          onClick={() => handleCheckoutCart(paymentMethod.code)}
+        >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)